src/sentry/incidents/logic.py

```diff
@@ -459,7 +460,16 @@ class AlertRuleNameAlreadyUsedError(Exception):
 
 # Default values for `SnubaQuery.resolution`, in minutes.
 DEFAULT_ALERT_RULE_RESOLUTION = 1
-DEFAULT_CMP_ALERT_RULE_RESOLUTION = 2
+DEFAULT_CMP_ALERT_RULE_RESOLUTION_MULTIPLIER = 2
+DEFAULT_ALERT_RULE_LOAD_SHEDDING_RESOLUTIONS = {
```

Comment:
Can we name this something with like `WINDOW_RESOLUTION_MAP` or something?

Reply:
hmm, i get that you'd want to change the name, but i'm not sure of that one.

I'd like it to have the same prefix, `DEFAULT_ALERT_RULE_` and include `RESOLUTION` to match the other variables. Generally i'm not a fan of including the data structure in the variable name like `map`, instead using things like `to` to illustrate what it's a map of.

 `DEFAULT_ALERT_RULE_WINDOW_TO_RESOLUTION` ?